### PR TITLE
Handle optional func args in tool calls when set to `None`

### PR DESCRIPTION
**Pull request by jayeshp19**
This PR fixes ValueError when optional function args are set to `None`

Related issues: 
https://github.com/livekit/agents/issues/1185
https://github.com/livekit/agents/issues/1208

### CODE REVIEW

**Review by davidzhao — APPROVED** [review]
nice

**Comment by davidzhao** [comment]
@jayeshp19 can you add a changeset?

**Comment by jayeshp19** [comment]
It's already added,  not sure why It's not detecting..

**Comment by jayeshp19** [comment]
retried and it detected

File: livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/_oai_api.py [hunk 1]
```diff
@@ -64,13 +64,19 @@ def create_ai_function_info(
             inner_type = typing.get_args(arg_info.type)[0]
```
Comment (mykola-mokhnach-parloa) [inline livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/_oai_api.py:64]: fetching the first argument does not work for type definitions like `Union[None, str]` or `None | str`, where the None is passed as the first argument. The validation should verify if arguments lengths is between 1 and 2 and then take the first matched non-None-type argument.

**Pull request merged**